jwst/dq_init/tests/test_dq_init.py

```diff
@@ -0,0 +1,340 @@
+import pytest
+from astropy.io import fits
+from jwst.dq_init import DQInitStep
+from jwst.dq_init.dq_initialization import do_dqinit
+from jwst.datamodels import MIRIRampModel, MaskModel
+from jwst.datamodels import GuiderRawModel, RampModel
+from jwst.datamodels import ImageModel, dqflags
+import numpy as np
+import numpy.testing as nptest
```

Comment:
flake8 says that `nptest` is also unused and hence can be removed.

Reply:
done